Formula/s/seclists.rb

```diff
@@ -0,0 +1,26 @@
+class Seclists < Formula
+  desc "Collection of multiple types of lists used during security assessments"
+  homepage "https://github.com/danielmiessler/SecLists"
+  url "https://github.com/danielmiessler/SecLists/tree/2024.1"
```

Comment:
This will just download the webpage. It needs to download the actual content. Check under the Releases in the repo for a tarbal link